requirements.txt

```diff
@@ -5,5 +5,5 @@ MarkupSafe==0.19
 WTForms==1.0.5
 Werkzeug==0.9.4
 itsdangerous==0.23
-pypuppetdb==0.1.1
+git+https://github.com/raphink/pypuppetdb@v4-api#egg=pypuppetdb-0.1.0
```

Comment:
Yes, I have another pr to fix for that and then I'll rebase. @mcanevet do you think you can fix the branch maybe?